api/auth.py

```diff
@@ -0,0 +1,59 @@
+import re
+
+from rest_framework import authentication, permissions
+from rest_framework import exceptions
+from django.conf import settings
+import jwt
+
+class CognomaAuthentication(authentication.BaseAuthentication):
+    def authenticate(self, request):
```

Comment:
That convention doesn't seem to be followed by any of the [Django Rest Framework docs](http://www.django-rest-framework.org/api-guide/authentication/)